conda_build/os_utils/liefldd.py

```diff
@@ -343,7 +345,9 @@ def _get_path_dirs(prefix):
 
 def get_uniqueness_key(file):
     binary = ensure_binary(file)
-    if binary.format == lief.EXE_FORMATS.MACHO:
+    if not binary:
+        return "unkown"
```

Comment:
```suggestion
        return lief.EXE_FORMATS.UNKNOWN
```
Wondering if that'd be okay as well?